SecurityPkg/Tcg/Tcg2Smm/Tcg2TraditionalMm.c

```diff
@@ -50,14 +52,34 @@ Tcg2NotifyMmReady (
   @retval FALSE This buffer is not valid per processor architecture or overlap with SMRAM.
 **/
 BOOLEAN
-IsBufferOutsideMmValid (
+Tcg2IsPrimaryBufferValid (
   IN EFI_PHYSICAL_ADDRESS  Buffer,
   IN UINT64                Length
   )
 {
   return SmmIsBufferOutsideSmmValid (Buffer, Length);
 }
 
+/**
+  This function is for the NonPrimary Buffer validation routine.
+  The NonPrimary Buffer is the buffer which is pointed from the
+  communication buffer.
+
+  @param Buffer  The buffer start address to be checked.
+  @param Length  The buffer length to be checked.
+
+  @retval TRUE  This buffer is valid.
+  @retval FALSE This buffer is not valid.
+**/
+BOOLEAN
+Tcg2IsNonPrimaryBufferValid (
+  IN EFI_PHYSICAL_ADDRESS  Buffer,
+  IN UINT64                Length
+  )
+{
+  return TRUE;
+}

```

Comment:
should you call "  return SmmIsBufferOutsideSmmValid (Buffer, Length);" here?

Reply:
originally, no such logic check for traditional SMM, so, i keep the TRUE here, but i agree we can provide the correct implemention here.